pkg/netceptor/netceptor.go

```diff
@@ -538,11 +544,58 @@ func (s *Netceptor) flood(message []byte, excludeConn string) {
 	}
 }
 
+// GetServerTLSConfig retrieves a server TLS config by name
```

Comment:
The client and server TLS config lists were previously global in scope, not scoped within a Netceptor object.   This should have been fixed long ago when Netceptor was updated to allow multiple instances.

Reply:
Awesome, this was something i've been worried about handling in the test suite LibNode stuff